build.gradle

```diff
@@ -108,7 +112,7 @@ ext.externalDependency = [
   "hadoopYarnClient": "org.apache.hadoop:hadoop-yarn-client:"+hadoopVersion,
   "hadoopYarnMiniCluster": "org.apache.hadoop:hadoop-minicluster:"+hadoopVersion,
   "hadoopAnnotations": "org.apache.hadoop:hadoop-annotations:"+hadoopVersion,
-  "hadoopAws": "org.apache.hadoop:hadoop-aws:2.6.0",
+  "hadoopAws": "org.apache.hadoop:hadoop-aws:"+hadoopAwsVersion,
```

Comment:
It shouldn't because it defaults:
`
if (!project.hasProperty('hadoopAwsVersion')) {
  hadoopAwsVersion = '2.6.0'
}
`